src/lib/expansion/expansion-panel-header.ts

```diff
@@ -101,6 +102,17 @@ export class MatExpansionPanelHeader implements OnDestroy {
     .subscribe(() => this._changeDetectorRef.markForCheck());
 
     _focusMonitor.monitor(_element.nativeElement, renderer, false);
+
+    // When a panel is hosted in an accordion, subscribe to the expand/collapse subject
+    if (this.panel.accordion) {
+      this._expandCollapseAllSubscription =
+        this.panel.accordion.expandCollapseAllSubject.subscribe((expanded: boolean) => {
+          // Only toggle if current state is not the intended state
+          if (!!this._isExpanded() !== expanded) {
+            this._toggle();
+          }
```

Comment:
Coupled with the move to `AccordionItem`, I would suggest just setting the expanded state directly rather than checking if a toggle needs to occur.


`this.panel.accordion.expandCollapseAllSubject.subscribe(expanded => this.expanded = expanded);`